source/faucet/main.d

```diff
@@ -48,6 +50,12 @@ private struct Config
 
     /// Between how many addresses we split a transaction by
     static immutable count = 15;
+
+    /// Bind address
+    public string address = "0.0.0.0";
+
+    /// Bind port
+    public ushort port = 0xACE;
```

Comment:
I think we usually use the decimal number for a port number, not hexadecimal. Don't we?

Reply:
Correct. But I saw `0xBOA` in agora :D